packages/concerto-core/api.txt

```diff
@@ -136,44 +136,44 @@ class TransactionDeclaration extends ClassDeclaration {
    + boolean hasInstance(object) 
 }
 class Concept extends Typed {
-   + boolean isConcept() 
+   + boolean _isConcept() 
 }
 class Identifiable extends Typed {
-   + string getIdentifier() 
-   + void setIdentifier(string) 
-   + string getFullyQualifiedIdentifier() 
+   + string _getIdentifier() 
+   + void _setIdentifier(string) 
+   + string _getFullyQualifiedIdentifier() 
    + String toString() 
-   + boolean isRelationship() 
-   + boolean isResource() 
-   + String toURI() 
+   + boolean _isRelationship() 
+   + boolean _isResource() 
+   + String _toURI() 
 }
 class Relationship extends Identifiable {
    + String toString() 
-   + boolean isRelationship() 
-   + Relationship fromURI(ModelManager,String,String,String) 
+   + boolean _isRelationship() 
+   + Relationship _fromURI(ModelManager,String,String,String) 
 }
 class Resource extends Identifiable {
    + String toString() 
-   + boolean isResource() 
-   + Object toJSON() 
+   + boolean _isResource() 
+   + Object _toJSON() 
```

Comment:
Intentional? JS calls `toJSON` automatically when you call JSON.stringify.

Reply:
Thanks. No this wasn't deliberate